fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -135,6 +135,31 @@ sealed interface ReviewQualityCheckState : State {
         }
     }
 
+    /**
+     * The source of the bottom sheet dismiss.
+     *
+     * @property sourceName Name of the dismiss source to be used in review quality check telemetry.
+     */
+    enum class DismissSource(val sourceName: String) {
```

Comment:
💡 This could be in a separate file with `BottomSheetViewState`, since it's not a state UI is observing